test-fixture/src/lib.rs

```diff
@@ -432,16 +432,16 @@ impl Write for SharedVec {
 }
 
 impl Display for SharedVec {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
         f.write_str(
             &String::from_utf8(
                 self.buf
                     .lock()
-                    .map_err(|_| std::fmt::Error)?
+                    .map_err(|_| fmt::Error)?
```

Comment:
For these, you could instead:

```rust
use std::fmt::Error as FmtError;
```

Not sure where I stand on that one though.

Reply:
Yeah, I thought about that. Not sure either which is better.